src/components/LoginModal/LoginModal.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from "react";
+import { Alert, Modal } from "react-bootstrap";
+
+const LoginModal = ({ setActiveModal, handleLogin, loginErr }) => {
+  const handleClose = () => {
+    setActiveModal("");
```

Comment:
You don't need to use `setActiveModal("")` inside compoents. You have `handleCloseModal` in `App` for that